src/main/java/namor/template/calculator/ClassicCalculator.java

```diff
@@ -0,0 +1,70 @@
+package namor.template.calculator;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+
+public class ClassicCalculator {
+
+    private LinkedList<String> operands;
+
+    public ClassicCalculator() {
+        this.operands = new LinkedList<>();
+    }
+
+    public double calculate(String arithmeticExpression) {
+
+        String[] operationElements = arithmeticExpression.split(" ");
+        operands.addAll(Arrays.asList(operationElements));
+
+        while(operands.size() > 1) {
+            doArithmeticOperation(operands.poll(), operands.poll(), operands.poll());
```

Comment:
PS:
This: `doArithmeticOperation(operands.poll(), operands.poll(), operands.poll());`
You can do like this: 
`String first = operands.poll();`
`String sign = operands.poll();`
`String second = operands.poll();`
`doArithmeticOperation(first, sign, second);`

yes few extra lines but at least in this way it's easy to understand what's going on in code